composer.json

```diff
@@ -24,12 +24,12 @@
         "doctrine/annotations": "^1.0",
         "doctrine/cache": "^1.0",
         "doctrine/collections": "^1.0",
-        "doctrine/event-manager": "^1.0",
-        "doctrine/reflection": "^1.2"
+        "doctrine/event-manager": "^1.0"
     },
     "require-dev": {
         "phpstan/phpstan": "^0.11",
         "doctrine/coding-standard": "^6.0",
+        "doctrine/common": "^3.0",
```

Comment:
So this is a circular dependency… yuck! It was a dev dependency in `doctrine/reflection`, not sure why… should I move it to a regular requirement?

Reply:
Can you check what will fail if this dependency is removed? A dev dependency of one package is not necessarily a dev dependency of the one that depends on it.

Reply:
I forgot to add it at first and it failed :slightly_smiling_face: